content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -140,14 +132,18 @@ $ kubectl label ns default istio.io/use-waypoint=waypoint
 namespace/default labeled
 {{< /text >}}
 
-After a namespace is enrolled to use a waypoint, any requests from any pods in ambient to any service running in that namespace will be routed through the waypoint for L7 processing and policy enforcement.
+After a namespace is enrolled to use a waypoint, any requests from any pods enrolled in the mesh to any service running in that namespace will be routed through the waypoint for L7 processing and policy enforcement.
```

Comment:
This is kind of misleading. Sidecar-enabled pods are part of the mesh but they won't use the waypoint...

Reply:
We stopped saying "in the ambient mesh" but this might be a reason to bring it back.
In a world where we have sidecars and waypoints, you would then need two policies - one with selectors and one with a `targetRef`?

Reply:
I retract my complaint. Not due to being invalid but I was simply way too in the weeds on a detail we don't support for 1.22 any way. Having said that there is potentially a weird caveat to consider for ingress gateways too. I suppose "pods enrolled in ztunnel" might help imply that ingress gateways are their own special case 🤷‍♂️